metabrainz/model/access_log.py

```diff
@@ -56,9 +57,9 @@ def create_record(cls, access_token, ip_address):
                     cls.token == access_token) \
             .count()
         if count > DIFFERENT_IP_LIMIT:
-            msg = ("Hourly access threshold exceeded for token %s\n\n"
+            msg = (gettext("Hourly access threshold exceeded for token %s\n\n"
```

Comment:
This doesn't need to be translated. It's a message for admin(s), who understand English.